src/platform/web.rs

```diff
@@ -27,3 +31,38 @@ impl WindowBuilderExtWebSys for WindowBuilder {
         self
     }
 }
+
+/// Additional methods on `EventLoop` to spawn the event loop without throwing an exception.
```

Comment:
```suggestion
/// Additional methods on `EventLoop` that are specific to the web.
```
I suppose the trait is "single-use" right now, but I don't believe we call that out in other single-platform extension traits.